pandas/core/series.py

```diff
@@ -2353,6 +2353,13 @@ def _gotitem(self, key, ndim, subset=None):
         return self
 
     _agg_doc = dedent("""
+    Notes
+    -----
```

Comment:
Can remove this section. Move the axis discussion to a substitution in parameters.